ts/components/messages/MessageAgenda.tsx

```diff
@@ -236,12 +266,73 @@ const FakeItemComponent = (
  */
 class MessageAgenda extends React.PureComponent<Props, State> {
   private sectionListRef = React.createRef<any>();
+  private overScrollAnim: Animated.Value;
+
+  public componentDidMount() {
+    this.state.scrollY.addListener(value => this.handleScroll(value));
+  }
+
+  public componentWillUnmount() {
+    this.state.scrollY.removeAllListeners();
+  }
+
+  public handleRelease() {
+    if (pot.isLoading(this.state.potAnim) && !pot.isSome(this.state.potAnim)) {
+      // start animation
+      this.setState({ potAnim: pot.some(this.state.potAnim) });
```

Comment:
```suggestion
      this.setState({ potAnim: pot.some(true) });
```
you are setting potAnim with the potAnim value. This is a fault and you can do it because of pot.Pot<any,any> declaration, this is why you should avoid any type